apix-impl/src/main/java/eu/xenit/apix/alfresco/search/SearchService.java

```diff
@@ -150,17 +152,16 @@ protected SearchParameters buildSearchParameters(SearchQuery postQuery) {
         return searchParameters;
     }
 
-    // These 2 methods are overridden in the 4.2 impl
     protected void setSearchLimit(SearchParameters searchParameters, int max) {
```

Comment:
To improve clarity, can we change the getter and setter to getMaxItems and setMaxItems?